queries.sql

```diff
@@ -1,99 +1,150 @@
+    GROUP BY species;
 
 /*------------------------------------------------------------*/
 /*What animals belong to Melody Pond?*/
-SELECT a.name FROM animals a JOIN owners o ON a.owner_id = o.id
-WHERE o.full_name = 'Melody Pond';
+    SELECT a.name FROM animals a JOIN owners o ON a.owner_id = o.id
+    WHERE o.full_name = 'Melody Pond';
 
 /*List of all animals that are pokemon (their type is Pokemon).*/
-SELECT a.name FROM animals a JOIN species s ON a.species_id = s.id
-WHERE s.name = 'Pokemon';
+    SELECT a.name FROM animals a JOIN species s ON a.species_id = s.id
+    WHERE s.name = 'Pokemon';
 
 /*List all owners and their animals, remember to include those that don't own any animal.*/
-SELECT o.full_name AS animals_owner, COALESCE(array_agg(a.name)) AS animals
-FROM owners o LEFT JOIN animals a ON o.id = a.owner_id GROUP BY o.id, o.full_name;
+    SELECT o.full_name AS animals_owner, COALESCE(array_agg(a.name)) AS animals
+    FROM owners o LEFT JOIN animals a ON o.id = a.owner_id GROUP BY o.id, o.full_name;
 
 /*How many animals are there per species?*/
-SELECT s.name, COUNT(a.id)FROM species s JOIN animals a ON s.id = a.species_id
-GROUP BY s.id, s.name;
+    SELECT s.name, COUNT(a.id)FROM species s JOIN animals a ON s.id = a.species_id
+    GROUP BY s.id, s.name;
 
 /*List all Digimon owned by Jennifer Orwell.*/
-SELECT a.name FROM animals a JOIN owners o ON a.owner_id = o.id
-JOIN species s ON a.species_id = s.id
-WHERE o.full_name = 'Jennifer Orwell' AND s.name = 'Digimon';
+    SELECT a.name FROM animals a JOIN owners o ON a.owner_id = o.id
+    JOIN species s ON a.species_id = s.id
+    WHERE o.full_name = 'Jennifer Orwell' AND s.name = 'Digimon';
 
 /*List all animals owned by Dean Winchester that haven't tried to escape.*/
-SELECT a.name FROM animals a
-JOIN owners o ON a.owner_id = o.id
-WHERE o.full_name = 'Dean Winchester' AND a.escape_attempts = 0;
+    SELECT a.name FROM animals a
+    JOIN owners o ON a.owner_id = o.id
+    WHERE o.full_name = 'Dean Winchester' AND a.escape_attempts = 0;
 
 /*Who owns the most animals?*/
-SELECT o.full_name, COUNT(a.id)FROM owners o
-JOIN animals a ON o.id = a.owner_id
-GROUP BY o.id, o.full_name ORDER BY COUNT(a.id) DESC
-LIMIT 1;
\ No newline at end of file
+    SELECT o.full_name, COUNT(a.id)FROM owners o
+    JOIN animals a ON o.id = a.owner_id
+    GROUP BY o.id, o.full_name ORDER BY COUNT(a.id) DESC
+    LIMIT 1;
+
+/*----------------------------------------------------------------*/
+
+/*Who was the last animal seen by William Tatcher?*/
+    SELECT a.name AS animal_name FROM visits v
+    JOIN animals a ON v.animal_id = a.id JOIN vets vt ON v.vet_id = vt.id
+    WHERE vt.name = 'William Tatcher'
+    ORDER BY v.visit_date DESC  LIMIT 1;
+
+/*How many different animals did Stephanie Mendez see?*/
+    SELECT COUNT(DISTINCT a.id) FROM visits v JOIN animals a ON v.animal_id = a.id
+    JOIN vets vt ON v.vet_id = vt.id WHERE vt.name = 'Stephanie Mendez';
```

Comment:
- [ ] Your result for this query as shown in your screenshot does not match, in your screenshot you have `3` but in the result of running your query the result is `4` please kindly add the correct result as your screenshot :+1: